plugins/flowdock/app/controllers/flowdock_controller.rb

```diff
@@ -0,0 +1,24 @@
+class FlowdockController < ApplicationController
+  rescue_from ActiveRecord::RecordNotFound do |error|
```

Comment:
Later in a another PR :) It would also be nice to have someone design a fancy 404 page for the html 404s.